backend/model/Submission.ts

```diff
@@ -0,0 +1,22 @@
+import mongoose, { Schema, Document, Types } from 'mongoose';
+
+export interface ISubmission extends Document {
+  email: string;
+  submissionDate: Date;
+  content: string;
+  numberOfUnitsTestPassed: Number;
+  testId: Types.ObjectId;
+  studentId: Types.ObjectId;
+}
+
+const SubmissionSchema: Schema = new Schema({
+  email: { type: String, required: true, unique: true },
+  submissionDate: { type: Date, required: true},
```

Comment:
```suggestion
  submissionTime: { type: Date, required: true, default: Date.now()},
```
ensure that when `Submission` document is created the submissionTime is equal to the time this document is created